alwayson/controllers/get.py

```diff
@@ -1,36 +1,27 @@
-from collections import defaultdict
-import json
-import os
+from typing import Annotated
 
-from fastapi import APIRouter 
+from fastapi import APIRouter, Depends
 from fastapi.requests import Request
 from fastapi.responses import JSONResponse
 
-router = APIRouter()
+from alwayson.controllers.injectors import inject_manifest_repo
```

Comment:
Just noticing casing differences - do pythonistas prefer CamelCase for classes and snake_case for the rest?

Reply:
Yea for Python Classes should be PascalCase and pretty much everything else is snake_case.